bundle/manifests/security-profiles-operator.clusterserviceversion.yaml

```diff
@@ -98,6 +98,7 @@ metadata:
       ]
     capabilities: Basic Install
     categories: Security
+    containerImage: registry.k8s.io/security-profiles-operator/security-profiles-operator:v0.6.0
```

Comment:
hmm, shouldn't this point at the `:latest` image we build in CI in between release and only point to the versioned image at the time of a release?

Reply:
I thought we're using the local image exclusively in the OLM tests, right?

Reply:
Yes, we do.
Sorry, I'm confused about this change (this hunk, the `replaces` part is fine). Why are we adding the `containerImage` directive? I couldn't find much about it in OLM docs..

Reply:
Ah, now I remember!
Thanks!